tests/dopamine/discrete_domains/run_experiment_test.py

```diff
@@ -443,6 +444,25 @@ def bundle_and_checkpoint(x, y):
     glob_string = '{}/events.out.tfevents.*'.format(self._test_subdir)
     self.assertGreater(len(tf.gfile.Glob(glob_string)), 0)
 
+  @parameterized.parameters([
+      (None, 4, 4), ((None, 1), 4, 1), ((-1, None), -4, -1)])
+  def testRewardClipping(self, reward_clipping, reward, expected_reward):
```

Comment:
The difficulty of writing this test reinforces my belief that clipping should be done in the agent (when implementing upstream).

Reply:
yes, I agree.